src/components/common/Waitlist/WaitlistCard.tsx

```diff
@@ -0,0 +1,67 @@
+import Image from "next/image";
+
+import {
+  Card,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+} from "~/components/ui/card";
+import { cn } from "~/lib/utils";
+
+type CardProperties = React.ComponentProps<typeof Card>;
+
+export type WaitlistCardProperties = CardProperties & {
+  cardTitle: string;
+  cardDescription: string;
+  cardIcon: string;
+};
+
+const waitlistBorderStyle = "border-[1px] border-[#525252] border-solid";
```

Comment:
use colors present in the tailwind config. dont use custom colors

Reply:
Done @JulianaSau